docs/index.rst

```diff
@@ -0,0 +1,68 @@
+.. _index:
+
+===================================================
+FTPVL Documentation
+===================================================
+FPGA Tool Performance Visualization Library (FTPVL) is a library for simplifying 
+the data collection, analysis, and visualization of performance metrics for
+Symbiflow development. Although it was made with Symbiflow in mind, it is highly
```

Comment:
`s/Symbiflow/SymbiFlow/g` - The `F` should be capitalized to be consistent with how it's spelled elsewhere.

Reply:
@TypingKoala Please apply this globally (i.e. search and replace.) I see some other instances of the incorrect spelling. Thanks.